bigquery/google/cloud/bigquery/job.py

```diff
@@ -2876,6 +2876,29 @@ def _format_for_exception(query, job_id):
 
         return template.format(job_id=job_id, header=header, ruler=ruler, body=body)
 
+    def _begin(self, client=None, retry=DEFAULT_RETRY):
+        """API call:  begin the job via a POST request
+
+        See
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs/insert
+
+        :type client: :class:`~google.cloud.bigquery.client.Client` or
+                    ``NoneType``
+        :param client: the client to use.  If not passed, falls back to the
+                    ``client`` stored on the current dataset.
```

Comment:
Dataset objects don't actually have `client` properties anymore, and haven't for a couple years. I've suggested some alternative language above. We should probably fix the docstring this was copied from, too.